server/db/helper.js

```diff
@@ -67,9 +67,18 @@ const getCourse = (id) => {
     });
 };
 
+const getUserBadges = (userId) => UserBadge.findAll({ 
+  where: {
+    id: userId,
```

Comment:
The badge's id doesn't match the userId, you are going to need to go through the UserBadge model first since that is our join table between users and badges